daisy/compute/compute.go

```diff
@@ -32,6 +30,10 @@ import (
 type Client struct {
 	hc  *http.Client
 	raw *compute.Service
+
+	OperationsWaitFake func(project, zone, name string) error
+	CreateDiskFake     func(project, zone string, d *compute.Disk) error
+	CreateImageFake    func(project string, i *compute.Image) error
```

Comment:
I don't think I like this, I don't like the idea of exposing testing features publicly. I much prefer the style of using a testClient